package.json

```diff
@@ -38,6 +42,7 @@
     "eslint": "^8.53.0",
     "husky": "^8.0.3",
     "lint-staged": "^15.1.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
if possible, would suggest switching to https://github.com/bcomnes/npm-run-all2, as it has a much lighter footprint but maintains the same api

[npm-run-all](https://pkg-graph.info/npm-run-all) vs [npm-run-all2](https://pkg-graph.info/npm-run-all2)

Reply:
Done in 47f55ac860e558d0d0d994cbfa1b19e89d62389b.